modules/virt-using-disk-image-install-windows.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-automating-windows-sysprep.adoc
+
+[id="virt-using-disk-image-install-windows_{context}"]
+= Using a disk image to install Windows
+
+Microsoft does not provide disk images for download, but you can create a disk image using the Windows DVD. This disk image can then be used to create virtual machines.
+
+.Procedure
+
+. Use the {VirtProductName} web console VM wizard to xref:../../virt/virtual_machines/virt-create-vms.adoc#virt-create-vms[create a new Windows VM], using the template for your desired Windows version.
```

Comment:
> `{VirtProductName}`

Is this supposed to be here? I am not familiar with adoc syntax; disregard if irrelevant.

Reply:
This displays as OpenShift Virtualization.